vnext/PropertySheets/JSEngine.props

```diff
@@ -14,12 +14,14 @@
     <HermesNoLink Condition="'$(HermesNoLink)' == '' and '$(Configuration)' == 'Release' and '$(EnableHermesInspectorInReleaseFlavor)' != 'true'">true</HermesNoLink>
     <!-- Use Hermes bytecode bundles provided by metro hermes compiler when available -->
     <EnableDevServerHBCBundles Condition="'$(EnableDevServerHBCBundles)' == ''">false</EnableDevServerHBCBundles>
+    <HermesPlatform Condition="'$(Platform)' == 'ARM64EC'">x64</HermesPlatform>
```

Comment:
I think we can get rid of this. @vmoroz has fixed the issue in hermes and we should be able to switch to arm64ec hermes.dll this upcoming week

Reply:
I'll merge as is for now and when the ARM64EC Hermes is available, a follow-up PR can be made just deleting this line.